blockchain/chain.go

```diff
@@ -1364,6 +1364,20 @@ func (b *BlockChain) BlockLocatorFromHash(hash *chainhash.Hash) BlockLocator {
 	return locator
 }
 
+func (b *BlockChain) IndexLookupNode(hash *chainhash.Hash) *blockNode {
```

Comment:
An exported function should always have a comment for documentation. https://go.dev/doc/comment